### ISSUE TITLE
Dimension Mismatch for encoder_hidden_state and face embeddings

### ISSUE THREAD

**Issue by Oguzhanercan** [MFaceTech/InstantID#6]
Hi, I get error for forward function of InstantIDAdapter class at torch.cat([encoder_hidden_states,face_embedding],dim=1)

when I print the shapes, encoder _hidden_states = 5,77,2048 and face_embedding shape is 1,16,2048. I did not change any hyperparameters, what would be wrong? @MFaceTech

**Comment by Oguzhanercan** [comment]
My bad, data loader problems.
